nominal/network/__init__.py

```diff
@@ -0,0 +1,3 @@
+from nominal.network.nominal_session import GzipRequestsClient, GzipRequestsSession
```

Comment:
We should probably throw this in a private module inside core? Or is there a use-case you were thinking of for keeping it outside of there?